src/pool/srv_pool.c

```diff
@@ -1566,7 +1566,7 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 	if (rc != 0)
 		goto out;
 
-	rc = init_svc_pool(svc, map_buf, map_version);
+	rc = init_svc_pool(svc, map_buf, map_version, svc->ps_rsvc.s_term);
```

Comment:
I suppose here the svc->ps_rsvc.s_term must be the up-to-date term? The ds_pool_iv_ns_update() call in pool_iv_pre_sync() and rebuild_tgt_prepare() could pass in stale term? In these two functions, why other operations with stale term are tolerable except the iv master rank updating?

Reply:
I suppose here the svc->ps_rsvc.s_term must be the up-to-date term? 
Yes,
The ds_pool_iv_ns_update() call in pool_iv_pre_sync() and rebuild_tgt_prepare() could pass in stale term?
Yes, so we add term check to make sure iv master rank will not be updated by stale term.

>In these two functions, why other operations with stale term are tolerable except the iv master rank updating?

Not sure I follow this question, could you please elaborate a bit?